civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.views_default.inc

```diff
@@ -2054,7 +2054,7 @@ return date(\'Y-m-d\');';
   $handler->display->display_options['header']['area_text_custom']['table'] = 'views';
   $handler->display->display_options['header']['area_text_custom']['field'] = 'area_text_custom';
   $handler->display->display_options['header']['area_text_custom']['empty'] = TRUE;
-  $handler->display->display_options['header']['area_text_custom']['content'] = '<a href="/node/add/hr-documents" class="btn btn-primary btn-lg"><i class="fa fa-plus-circle"></i>&nbsp;Add HR Document</a>';
+  $handler->display->display_options['header']['area_text_custom']['content'] = '<a href="/node/add/hr-documents" class="btn btn-primary btn-lg"><i class="fa fa-plus-circle"></i>&nbsp;Add HR Resource</a>';
```

Comment:
I know this an exported view, and that in a header is difficult to write code to add a link, but I think this code could generate problems for installs which its base it not a domain, for example inside CiviHR inside a folder.
Is there a better way to do this?